packages/metal-quartz-checkbox/src/MetalQuartzCheckbox.js

```diff
@@ -19,6 +19,53 @@ class MetalQuartzCheckbox extends Component {
  * @type {!Object}
  */
 MetalQuartzCheckbox.STATE = {
+	checkboxInline: {
```

Comment:
Hey @pat270,

From the name, it's not clear to me what this attribute does. So this is a list of objects with props for each checkbox?